src/components/Notifications/form/notification.ts

```diff
@@ -162,7 +162,7 @@ export const getNotificationSchema = (
       "ui:data-cy": "jira-comment-input",
     },
     slackInput: {
-      "ui:placeholder": "@user or #channel",
+      "ui:placeholder": "MEMBERID, @user or #channel",
```

Comment:
```suggestion
      "ui:placeholder": "Member ID, @user or #channel",
```

Reply:
I wasn't sure about this one. We tend to display them with hints for the format, such as @user and #channel. The member ID will look something like this: UJKMY34TL. I was trying to find a way to hint that to the user. Any ideas?

Reply:
Oooh, that sounds like a good candidate for some description text maybe? Something like `Notifications can be sent to a Slack channel, @user, or member ID represented as an alphanumeric string.`?